test/k8sT/Policies.go

```diff
@@ -293,6 +293,11 @@ var _ = SkipDescribeIf(func() bool {
 				helpers.CurlFail(fmt.Sprintf("http://%s/public", clusterIP)))
 			res.ExpectFail("%q can curl to %q", appPods[helpers.App3], clusterIP)
 
+			_, err = kubectl.CiliumPolicyAction(
+				namespaceForTest, l3Policy,
+				helpers.KubectlDelete, helpers.HelperTimeout)
+			Expect(err).Should(BeNil(), "Cannot delete L3 Policy")
```

Comment:
I'm not sure this change is correct. If we don't have a L3 allow policy for the L3 deny test, then everything is denied and we're not really testing the deny policy. /cc @aanm 

(FYI, if we keep this, it should be in a separate commit as AFAIU, it's independent from the change above.)

Reply:
Correct. We shouldn't remove the L3 allow policy. Well, it might make sense to test without a L3 allow policy and with a L3 allow policy. On both cases the traffic should be dropped.

Reply:
I see. I'll fix it :)